scrypto/src/crypto/ecdsa.rs

```diff
@@ -237,7 +251,7 @@ impl FromStr for EcdsaPrivateKey {
 
 impl fmt::Display for EcdsaPrivateKey {
     fn fmt(&self, f: &mut fmt::Formatter) -> Result<(), fmt::Error> {
-        write!(f, "{}", hex::encode(self.0))
+        write!(f, "{}", hex::encode(self.to_vec()))
```

Comment:
Could we also remove this? :)

Reply:
What's the problem with this?

Reply:
Printing private keys?

Reply:
I feel like were are being paternalistic.

If they want to print private keys, they know what they're doing (we dump private key to screen in simulator; Metamask displays private key on screen). 

On the other hand, I understand your standpoint. Disallowing all insecure operations.

Reply:
even if only by accident in this case, e.g. if EcdsaPrivateKey is some struct which wants to be printed. I think need to make interface such that it's "hard" to display but possible.